docs/src/components/authrequest.jsx

```diff
@@ -0,0 +1,453 @@
+              />
+              <label className="ml-4" htmlFor={`scope_${scope}`}>
+                {scope}{" "}
+                {scopeIndex === 8 && scopeState[8] && !organizationId ? (
+                  <strong className="text-red-500">
+                    Organization ID missing!
+                  </strong>
+                ) : null}
+              </label>
+            </div>
+          );
+        })}
+      </div>
+
+      {/* <h5>Optional Parameters</h5>
+
+      <div className={styles.grid}>
+        <div className={styles.inputwrapper}>
+          <label className={styles.label}>Id Token Hint</label>
+          <input
+            className={styles.input}
+            id="id_token_hint"
+            value={idTokenHint}
+            onChange={(event) => {
+              const value = event.target.value;
+              setIdTokenHint(value);
+            }}
+          />
+        </div>
+      </div> */}
+
+      <br />
+
+      <div className="rounded-md bg-gray-700 shadow dark:bg-black/10 p-2 flex flex-col items-center">
+        <code className="text-sm w-full mb-4 bg-transparent border-none">
+          <span className="text-yellow-500">{instance}</span>
+          <span className="text-white">oauth/v2/authorize</span>
+          <CodeSnipped cname="text-green-500">{`?client_id=${encodeURIComponent(
+            clientId
+          )}`}</CodeSnipped>
+          <CodeSnipped cname="text-blue-500">{`&redirect_uri=${encodeURIComponent(
+            redirectUri
+          )}`}</CodeSnipped>
+          <CodeSnipped cname="text-orange-500">
+            {`&response_type=${encodeURIComponent(responseType)}`}
+          </CodeSnipped>
+          <CodeSnipped cname="text-purple-500">{`&scope=${encodeURIComponent(
+            scope
+          )}`}</CodeSnipped>
+          {prompt && (
+            <CodeSnipped cname="text-cyan-500">{`&prompt=${encodeURIComponent(
+              prompt
+            )}`}</CodeSnipped>
+          )}
+          {loginHint && prompt === "select_account" && (
+            <CodeSnipped cname="text-rose-500">{`&login_hint=${encodeURIComponent(
+              loginHint
+            )}`}</CodeSnipped>
+          )}
+        </code>
+
+        <a
+          target="_blank"
+          className="mt-2 flex flex-row items-center py-2 px-4 text-white bg-green-500 dark:bg-green-600 hover:dark:bg-green-500 hover:text-white rounded-md hover:no-underline font-semibold text-sm"
+          href={`${instance}oauth/v2/authorize?client_id=${encodeURIComponent(
+            clientId
+          )}&redirect_uri=${encodeURIComponent(
+            redirectUri
+          )}&response_type=${encodeURIComponent(
+            responseType
+          )}&scope=${encodeURIComponent(scope)}${
+            prompt ? `&prompt=${encodeURIComponent(prompt)}` : ""
+          }${
+            loginHint && prompt === "select_account"
+              ? `&login_hint=${encodeURIComponent(loginHint)}`
+              : ""
+          }`}
+        >
```

Comment:
Add plausible

Reply:
done but has to be tested as we have no other window.plausible(...) implemented